cypress/support/generate.js

```diff
@@ -0,0 +1,14 @@
+import { faker } from '@faker-js/faker';
+
+function generateUser() {
+  const username = faker.internet.userName();
+  const firstName = faker.person.firstName();
+  const lastName = faker.person.lastName();
+  const phone = faker.phone.number();
+  const address = faker.location.streetAddress();
+  const email = `${username}@mail.com`;
+
+  return { email, firstName, lastName, phone, address };
+}
+
+module.exports = { generateUser };
```

Comment:
Since you're using ES6 syntax for importing modules, it would be more consistent to use ES6 syntax for exporting as well. Instead of 'module.exports = { generateUser };', you can use 'export { generateUser };'.